scalableminds/webknossos
# Fix incorrect rotation matrix after selecting nodes

Nodes that are created in arbitrary mode have an attached non-default rotation. Selecting that node will center their position and change apply their rotation. In orthogonal mode, this shouldn't happen as that rotation is used for bucket picking.

### URL of deployed dev instance (used for testing):
- https://fixchangingrotation.webknossos.xyz

### Steps to test:
- switch to flight mode with non-default angles (e.g., 45**3)
- create some nodes
- switch back to ortho mode and select nodes
- data should be rendered without problems

### Issues:
- second fix for #5288 

------
(Please delete unneeded items, merge only when none are left open)
